static/src/stylesheets/module/commercial/_advert.scss

```diff
@@ -126,9 +126,9 @@
     display: flex;
     align-items: stretch;
 
-    > * {
-        width: 100%;
-    }
+    // > * {
+    //     width: 100%;
+    // }
```

Comment:
This seems to be contributing to layout difficulties on the paidFor containers, removing it hasn't broken anything AFAIK... _yet_

🔥⁉️